hack/update-bazel.sh

```diff
@@ -31,9 +29,8 @@ export GOBIN="${KUBE_OUTPUT_BINPATH}"
 PATH="${GOBIN}:${PATH}"
 
 # Install tools we need, but only from vendor/...
-go install ./vendor/github.com/bazelbuild/bazel-gazelle/cmd/gazelle
-
-go install ./vendor/github.com/kubernetes/repo-infra/kazel
+go install k8s.io/kubernetes/vendor/github.com/bazelbuild/bazel-gazelle/cmd/gazelle
```

Comment:
for consistency, should we update https://github.com/kubernetes/kubernetes/blob/7f23a743e8c23ac6489340bbb34fa6f1d392db9d/hack/verify-spelling.sh#L27-L28 too?

Reply:
SGTM